src/gui/interface/Window.h

```diff
@@ -1,11 +1,15 @@
 #ifndef WINDOW_H
 #define WINDOW_H
-
+#include "gui/interface/Button.h"
+#include "gui/interface/Label.h"
+#include "graphics/Graphics.h"
+#include <stdio.h>
 #include <string>
 #include <vector>
 #include "Config.h"
 #include "gui/interface/Point.h"
-
+#include <iostream>
+#include <math.h>
```

Comment:
This should probably be `#include <cmath>`

Reply:
If I merged it I would probably fix the minor formatting details myself. (but I definitely could list tons of minor fixes similar to this. I like to include standard libraries first, so technically these should go up a few lines)